Monitored notifications

gavinwye opened the pull request:
This pull request contains the pages for monitored notifications.

The journey is as follows:

## Emails
All email will detail transactions that happened the previous day. 
For example, an email that is sent on the 02/MM/YYYY will list transactions for 01/MM/YYYY
- [Day 1 - intro email](https://pm-intelligence-tool-pr-33.herokuapp.com/notifications/emails/intro-daily)
Potentially delivered to 60 users who have logged in at least once a week for 12 weeks out of the last 25 weeks and are monitoring at least one thing. Day 1 will not be the same date for all users as they will need to be monitoring a prisoner/payment source that has done something before they get the day 1 email.
- [Day 1 - intro email not monitoring](https://pm-intelligence-tool-pr-33.herokuapp.com/notifications/emails/intro-daily-not-monitoring)
Potentially Delivered to 12 users who are not monitoring anything but have logged in at least once a week for 20 weeks out of the last 25 weeks
- [Day 2+ - email with notifications ](https://pm-intelligence-tool-pr-33.herokuapp.com/notifications/emails/daily)
Only sent to users who are monitoring things that have had transactions in the previous day.

## Notification page
The notifications page will update when we have new notifications in the same way that the monitored tiles on the homepage do.
- [Notifications for things users are monitoring](https://pm-intelligence-tool-pr-33.herokuapp.com/notifications/) 
- [Notifications if users have never monitored anything ](https://pm-intelligence-tool-pr-33.herokuapp.com/notifications/not-monitoring)
- [Notifications if users have stopped following things](https://pm-intelligence-tool-pr-33.herokuapp.com/notifications/stopped-monitoring)

Pagination will be displayed once there are last 25 days worth of results. We may need to refine this once we see real data in the build.

## Setting page
- [This page shows the notifications off and on](https://pm-intelligence-tool-pr-33.herokuapp.com/settings)

gavinwye committed:
New version of the notifications page for monitored things

gavinwye committed:
Changed the setting message so that it’s focussed towards monitoring

gavinwye committed:
Added a prisoner page

gavinwye committed:
modified the notifications page

gavinwye committed:
added email notifications

gavinwye committed:
Spelling corrections

ushkarev commented:
looks good :)

one thing i'd note is that in the first "day 1" section, where it says delivered to 60 people really it means _could_ be delivered to 60 people. cos they might not have any notifications on what they monitor

gavinwye commented:
Good point @ushkarev 👍I've updated the description to make that more clear

gavinwye committed:
Content changes for notifications

gavinwye commented:
I've made all of the content design changes now. Just need to include pagination and possibly put some real results in from a json file then this is done.

janestead committed:
Update intro-daily.html

Amended the heading, settings screen line and the sign off 'Prisoner money team'

gavinwye committed:
Merge branch 'master' into monitored-notifications

janestead committed:
updated content

updated content as in google doc

janestead committed:
updated content

changes as per comments

janestead committed:
feedback link

tried to write in this ^^ and the settings screen link

gavinwye committed:
added payment source name

gavinwye committed:
removed old pagination files

gavinwye committed:
Merge branch 'monitored-notifications' of github.com:gavinwye/prisoner-money-intelligence-tool-prototype into monitored-notifications

gavinwye committed:
removed redundant notifications files

gavinwye committed:
Added the MOJ frontend

gavinwye committed:
Added pagination from the MOJ design system

gavinwye committed:
removed duplicate heading

gavinwye committed:
Changed contnet for email notifications settings
